### PR TITLE
fix pip & Kraken trade history. added ledger info

### PR DESCRIPTION
**Pull request by cupidsdead**
fixes pipfile dependency break
fixes get_trades_history

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -26,7 +26,7 @@ cchardet = "*"              # cchardet is a faster replacement for chardet
 aiofile = ">=2.0.0"         # used by util/async_file.py
 aiokafka = "*"              # pip install cryptofeed[kafka]
 hiredis = "*"               # pip install cryptofeed[redis]
-aioredis = ">=2.0.0a1"      # pip install cryptofeed[redis]
+aioredis = "*"      # pip install cryptofeed[redis]
```
Comment (bmoscon) [inline Pipfile:29]: why did you change this? it will only work with that specific version of aioredis (the pre-release alpha). Older versions will not work.

**Review by bmoscon — CHANGES_REQUESTED** [review]

File: Pipfile (hunk 1 above)
Comment (cupidsdead) [reply Pipfile:29]: I see. On that version it was unable to resolve dependencies and create a lockfile via Pipfile.

File: cryptofeed/rest/kraken.py [hunk 2]
```diff
@@ -160,7 +160,8 @@ def trades(self, symbol: str, start=None, end=None, retry=None, retry_wait=10):
             if not end:
                 end = pd.Timestamp.utcnow()
             for data in self._historical_trades(symbol, start, end, retry, retry_wait):
-                yield list(map(lambda x: self._trade_normalization(x, symbol), data['result'][next(iter(data['result']))]))
+                yield list(
+                    map(lambda x: self._trade_normalization(x, symbol), data['result'][next(iter(data['result']))]))
```
Comment (bmoscon) [inline cryptofeed/rest/kraken.py:164]: looks like just a formatting change, please undo

File: cryptofeed/rest/kraken.py [hunk 3]
```diff
@@ -317,3 +281,91 @@ def cancel_order(self, order_id: str):
...
+            XETH => ETH
+            ZGBP => GBP
+        """
+        cleansym = sym
+        try:
+            if len(sym) == 8:
+                cleansym = sym[1:4] + sym[5:]
+            elif len(sym) == 4:
+                cleansym = sym[1:]
+        except Exception as ex:
```
Comment (bmoscon) [inline cryptofeed/rest/kraken.py:368]: at a minimum log this, i can't think of any case where an except pass is a good idea, except on perhaps places where you might be trying to import various modules and might expect issues

**Review by bmoscon — CHANGES_REQUESTED** [review]
looks pretty good, just a few minor changes :)

File: cryptofeed/rest/kraken.py (hunk 3 above)
Comment (cupidsdead) [reply cryptofeed/rest/kraken.py:368]: OK this is now being logged and expanded the logic so we catch pairs like ETHUSDT

**Comment by cupidsdead** [comment]
> looks pretty good, just a few minor changes :)

Thanks for checking :) have made your requested changes. are we able to merge now? Cheers.

File: cryptofeed/rest/kraken.py (hunk 2 above)
Comment (cupidsdead) [reply cryptofeed/rest/kraken.py]: this is now resolved

**Review by bmoscon — APPROVED** [review]
